src/views/Player.vue

```diff
@@ -15,6 +15,7 @@
           :ref="'position' + plioQuestion.id.toString()"
           :isTutorialComplete="isTutorialComplete"
           :tutorialProgress="tutorialProgress"
+          :progressBarInfo="progressBarInfo"
```

Comment:
sending the progress bar info to `PlioQuestions.vue`

Reply:
As above, don't send to `PlioQuestion`. I don't know how complicated it is, though because I can't think reading through the PR how it will work.... animation should be handled by the progress bar and player. Maybe the right answer is to have a meta-component for the modal, which will have two (or more in future) components: `PlioQuestion`, `ProgressBar`, and this meta-component can deal with both things.